pkgs/development/idris2-modules/idris2-compiler.nix

```diff
@@ -51,6 +53,8 @@ in stdenv.mkDerivation rec {
   checkInputs = [ gambit nodejs ]; # racket ];
   checkFlags = [ "INTERACTIVE=" ];
 
+  outputs = [ "out" ] ++ vendored-packages;
+
   # TODO: Move this into its own derivation, such that this can be changed
   #       without having to recompile idris2 every time.
   postInstall = let
```

Comment:
Should we add `prelude` (and maybe others?) to the global libraries? Or maybe don't take them out of $out in the first place? Without prelude, the interpreter won't even start.